includes/blocks.inc

```diff
@@ -422,7 +422,7 @@ function islandora_solr_simple_search_form($form, &$form_state) {
   $form['simple']["islandora_simple_search_query"] = array(
     '#size' => '15',
     '#type' => 'textfield',
-    '#title' => '',
+    '#title' => 'Search For:',
```

Comment:
I decided to use "Search term", while I think you're right that technically they're different, interpreting WCAG Guideline 3.2: "Make Web pages appear and operate in predictable ways.", I think from the end user, it fits better this way...

I've added a note to the Pull request above that should make the visibility of the new label more explicit.